content/en/blog/_posts/2021-12-15-pod-security-admission-beta.md

```diff
@@ -0,0 +1,778 @@
+
+## Why Pod Security
+
+Pod Security overcomes key shortcomings of Kubernetes' existing, but deprecated, PodSecurityPolicy (PSP) mechanism:
+
+   * Policy authorization model — challenging to deploy with controllers.
+   * Risks around switching — a lack of dry-run/audit capabilities made it hard to enable PodSecurityPolicy.
+   * Inconsistent and Unbounded API — the large configuration surface and evolving constraints led to a complex and confusing API.
+
+The shortcomings of PSP made it very difficult to use which led the community to reevaluate whether or not a better implementation could achieve the same goals. One of those goals was to provide an out-of-the-box solution to apply security best practices. Pod Security ships with predefined Pod Security levels that a cluster administrator can configure to meet the desired security posture.
+
+It's important to note that Pod Security doesn't have complete feature parity with the deprecated PodSecurityPolicy. Specifically, it doesn't have the ability to mutate or change Kubernetes resources to auto-remediate a policy violation on behalf of the user. Additionally, it doesn't provide fine-grained control over each allowed field and value within a pod specification or any other Kubernetes resource that you may wish to evaluate. If you need more fine-grained policy control then take a look at these [other](/docs/concepts/security/pod-security-standards/#faq) projects which support such use cases.
+
+Pod Security also adheres to Kubernetes best practices of declarative object management by denying resources that violate the policy. This requires resources to be updated in source repositories, and tooling to be updated prior to being deployed to Kubernetes.
+
+## How Does Pod Security Work?
+    
+Pod Security is a built-in [admission controller](/docs/reference/access-authn-authz/admission-controllers/) starting with Kubernetes v1.22, but can also be run as a standalone [webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/). Admission controllers function by intercepting requests in the Kubernetes API server prior to persistence to storage. They can either `admit` or `deny` a request. In the case of Pod Security, pod specifications will be evaluated against a configured policy in the form of a Pod Security Standard. This means that security sensitive fields in a pod specification will only be allowed to have [specific](h/docs/concepts/security/pod-security-standards/#profile-details) values. 
```

Comment:
Is there a better place for “webhook” to hyperlink to / will there be, fairly soon?

Reply:
https://github.com/kubernetes/website/blob/dev-1.23/content/en/docs/concepts/security/pod-security-admission.md#alternative-installing-the-podsecurity-admission-webhook is in the 1.23 branch (the anchor isn't very well-named though)